frontend/public/components/monitoring/alerting.tsx

```diff
@@ -585,6 +596,78 @@ const alertStateToProps = (state: RootState, { match }): AlertsDetailsPageProps
   };
 };
 
+const SilenceTableRow: RowFunction<Silence> = ({ index, key, obj, style }) => {
```

Comment:
Could you please return `SilenceTableRow` to its original place in the file? That will make it easier to read the diff / track changes in the git history.

Similarly for `AlertTableRow`, `ActiveAlerts`, `AlertStateDescription`, anything else?